deploy/helm/values.yaml

```diff
@@ -63,10 +63,14 @@ redis:
 # Find out more about configuration options at: https://github.com/bitnami/charts/tree/main/bitnami/mongodb/#parameters
 #
 mongodb:
-  #externalUrl:
   enabled: true
   service:
     nameOverride: lowcoder-mongodb
+    #externalUrl:
```

Comment:
Moved externalUrl under `service` since this is how it's referenced [here](https://github.com/lowcoder-org/lowcoder/blob/main/deploy/helm/templates/api-service/secrets.yaml#L8)

Reply:
Thank you for catching it.